add path to BamTools header files, otherwise some auxiliary programs don't compile
author: Albert Bogdanowicz
diff -Nru augustus.orig/auxprogs/bam2hints/Makefile augustus/auxprogs/bam2hints/Makefile
--- augustus.orig/auxprogs/bam2hints/Makefile	2016-03-30 16:25:29.000000000 +0200
+++ augustus/auxprogs/bam2hints/Makefile	2018-04-12 16:04:16.212990536 +0200
@@ -8,7 +8,7 @@
 #	Last modified:  09-October-2015 by Katharina J. Hoff
 
 # Variable definition
-INCLUDES = /usr/include/bamtools
+INCLUDES = $(EBROOTBAMTOOLS)/include/bamtools
 LIBS = -lbamtools -lz
 SOURCES = bam2hints.cc 
 OBJECTS = $(SOURCES:.cc=.o)
diff -Nru augustus.orig/auxprogs/filterBam/src/Makefile augustus/auxprogs/filterBam/src/Makefile
--- augustus.orig/auxprogs/filterBam/src/Makefile	2016-04-20 16:32:58.000000000 +0200
+++ augustus/auxprogs/filterBam/src/Makefile	2018-04-12 16:04:43.033325771 +0200
@@ -9,7 +9,7 @@
 PROGRAM = filterBam
 OBJECTS = $(SOURCES:.cc=.o)
 BAMTOOLS = /usr/include/bamtools
-INCLUDES = -I$(BAMTOOLS) -Iheaders -I./bamtools
+INCLUDES = -I$(BAMTOOLS) -Iheaders -I./bamtools -I$(EBROOTBAMTOOLS)/include/bamtools
 LIBS = -lbamtools -lz
 CFLAGS = -std=c++0x
 VPATH = functions

